scalafix-tests/unit/src/test/scala-target2/scalafix/tests/v1/SymbolSuite.scala

```diff
@@ -22,6 +22,9 @@ class SymbolSuite extends munit.FunSuite {
       case Term.ApplyInfix(_, Term.Name("shouldBe"), _, arg :: Nil) => arg
     }
 
+    // The symbol lookup fails against Scala 3.1.1 SemanticDB as the position
+    // there excludes surrounding parentheses while 2.x (scalac-semanticdb) and
+    // the parser include them
```

Comment:
I am not familiar with changes in Scala3, but I know there has been changes to the infix behavior. Where should we report this mismatch between scalameta's parser and dotty's semanticdb output? Is it something that should be addressed in Dotty, in Scalameta, or here?